index.js

```diff
@@ -2,17 +2,39 @@
 //.
 //. [Fluture][] type definitions for [Sanctuary][].
 //.
+//. ## Usage
+//.
 //. ```console
-//. $ npm install --save fluture sanctuary-def fluture-sanctuary-types
+//. $ npm install --save fluture-sanctuary-types
 //. ```
 //.
-//. ## Usage
+//. Note that you also need [Fluture][] and [sanctuary-def][] installed.
+//. Sanctuary-def comes preinstalled with Sanctuary, so you could install
+//. either one. Fluture has to be installed separately. See `package.json`
+//. for compatible versions (defined in `peerDependencies`).
+//.
+//. ### EcmaScript Module
 //.
 //. ```js
-//. import $ from 'sanctuary-def';
```

Comment:
Should this line be kept, given the use of `$.Number`?

Reply:
Ah, darn it